mysite/missions/templates/missions/pipvirtualenv/setup.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'missions/pipvirtualenv/base.html' %}
+{% comment %}
+# This file is part of OpenHatch.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+
+{% block mission_main %}
+<div class='submodule fat'>
+  <div class='head'>
+    <h3>Setup pip and virtualenv</h3>
+  </div>
+  <div class="body">
+    <p>We are going to use a shell script(for Linux) and a Mac application
+    which will automatically download and install pip and virtualenv on our
+    system. The shell script and the Mac application are part of a project
+    called <a href="http://pip2014.com/">pip2014</a>, a project by 
+    <a href="https://glyph.twistedmatrix.com/">Glyph</a>.</p>
+
+    <h3>Linux</h3>
+
+    <p>Open up the terminal, type the following command and press Enter:</p>
+    <pre>curl https://glyph.im/pip/bootstrap.sh | bash</pre>
+
+    <p>Some Python packages that have C extensions also require a C compiler
+    so that the C extensions can be compiled. A good example of a package
+    that has such requirements is <a href="http://www.numpy.org/">NumPy</a>,
+    a package for scientific computing with Python.</p>
+
+    <p>To install a C compiler on Debian or Ubuntu, at the terminal,
+    type the following command and press Enter:</p>
+    <pre>sudo apt-get install build-essential python-dev</pre>
+
+    <p>To install a C compiler on Fedora, at the terminal, type the
+    following command and press Enter:</p>
+    <pre>sudo yum install python-devel</pre>
+
```

Comment:
@eeshangarg The Python Guide http://docs.python-guide.org/en/latest/starting/install/osx/ has info under the Doing It Right section. The second paragraph and the following note are the important points to make (or just refer to the Python Guide).